docs/BEMAN_STANDARD.md

```diff
@@ -94,6 +94,22 @@ Examples: A `beman.smart_pointer` library's repository should be named `smart_po
 **[REPOSITORY.CODEOWNERS]** REQUIREMENT: There must be a `.github/CODEOWNERS` file
 with a relevant set of codeowners.
 
+**[REPOSITORY.DISALLOW_GIT_SUBMODULES]** RECOMMENDATION: The repository should not use git submodules. All its dependencies should be fetched via [cmake FetchContent](https://cmake.org/cmake/help/latest/module/FetchContent.html).
```

Comment:
I wonder if [REPOSITORY.DISALLOW_GIT_SUBMODULES] should be distinct from a separate [CMAKE.USE_FETCHCONTENT] rule.

Reply:
Agree. I think I would also prefer your approach.

Reply:
I don't know if there is a reasonable alternative for submodule  for the https://github.com/mpark/wg21 framework? 
Since I think git submodules are the wrong tool for a job, I'd like to find a better solution, but I don't think FetchContent is an alternative. We're saying should, not must, here, so I think it's ok, but perhaps clarify that submodules are to be avoided, not forbidden?

Reply:
I tried to apply both suggestions.
* split in to rules
* add an explicit `known exception` for https://github.com/mpark/wg21